backend/python/app/tests/helpers/story_helpers.py

```diff
@@ -0,0 +1,23 @@
+from ...models.story import Story
+
+
+def assert_story_equals_model(story_response, story_model, graphql_response=True):
+    assert story_response["title"] == story_model.title
+    assert story_response["description"] == story_model.description
+    assert story_response["level"] == story_model.level
+
+    if graphql_response:
+        assert story_response["youtubeLink"] == story_model.youtube_link
+        # TODO: expected translatedLanguages?
+        assert story_response["translatedLanguages"] == None
+    else:
+        assert story_response["youtube_link"] == story_model.youtube_link
+        # TODO: expected translated_languages?
+        assert story_response["translated_languages"] == None
+
+
+def require_story_in_db(db):
```

Comment:
nit: add a quick comment explaining what this function does

Reply:
renamed to a more descriptive `guarantee_story_table_not_empty`